content/spin/v2/triggers.md

```diff
@@ -13,9 +13,70 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/triggers.md"
 
 A Spin _trigger_ maps an event, such as an HTTP request or a Redis pub-sub message, to a component that handles that event.
 
-An application can contain multiple triggers, but they must all be the same type. For example, an application can contain triggers for multiple HTTP routes, or for multiple Redis pub-sub channels, but can't contain both an HTTP _and_ a Redis trigger.
+An application can contain multiple triggers. As of Spin 2.3 onward, a Spin application can use any combination of trigger types within a single application. For example, a single application can contain triggers for both HTTP routes and Redis pub-sub channels. 
```

Comment:
Incidentally, even in 2.3, multi-trigger-type is an experimental feature with some rough edges...